packages/ui/src/pages/Consent/index.tsx

```diff
@@ -1,11 +1,20 @@
-import React, { useEffect } from 'react';
+import React, { useEffect, useContext } from 'react';
+import { useTranslation } from 'react-i18next';
 
 import { consent } from '@/apis/consent';
+import BrandingHeader from '@/components/BrandingHeader';
 import useApi from '@/hooks/use-api';
+import { PageContext } from '@/hooks/use-page-context';
+
+import * as styles from './index.module.scss';
 
 const Consent = () => {
+  const { experienceSettings } = useContext(PageContext);
+  const { logoUrl = '' } = experienceSettings?.branding ?? {};
```

Comment:
Just curious, do we have different logos for both light and dark themes?

Reply:
Agree. Could leave this to Eric